phpBB/phpbb/db/driver/mssql_odbc.php

```diff
@@ -156,7 +158,7 @@ function sql_query($query = '', $cache_ttl = 0)
 			{
 				$this->sql_report('start', $query);
 			}
-			else if (defined('PHPBB_DISPLAY_LOAD_TIME'))
+			else if ($phpbb_container->getParameter('debug.load_time'))
```

Comment:
like for your other PR. I think it could be a parameter that we set during a boostrap phase once the container is built

Reply:
But how? How do we receive the value which we want to be stored inside yml file?

Reply:
a setter in the class and an after boot step that : 
1) retrieve the parameter value from the container
2) retrieve the service from the container
3) call the service setters with the parameter value

Reply:
Yeah true. But the problem is more fundamental. SQL query is called two times before container is built (without cache). `sql_query()`, however, uses `debug.load_time`. So first 2 queries wouldn't have SQL time calculated.

Reply:
@Nicofuma like this?

Reply:
something like that yes